pkg/cmd/run/rerun/rerun.go

```diff
@@ -39,8 +40,19 @@ func NewCmdRerun(f *cmdutil.Factory, runF func(*RerunOptions) error) *cobra.Comm
 
 	cmd := &cobra.Command{
 		Use:   "rerun [<run-id>]",
-		Short: "Rerun a failed run",
-		Args:  cobra.MaximumNArgs(1),
+		Short: "Rerun a run",
+		Long: heredoc.Docf(`
+			Rerun an entire run, only failed jobs, or a specific job from a run.
+
+			Note that due for historical reasons, the %[1]s--job%[1]s flag may not take what you expect.
+			Specifically, when navigating to a job in the browser, the URL looks like this:
+			%[1]shttps://github.com/<org>/<repo>/actions/runs/<run-id>/jobs/<job-id>%[1]s.
```

Comment:
Maybe we should call the placeholder something other than `<job-id>`  to avoid having the reader assume that this is an actual job ID. Something vague could be better
```suggestion
			%[1]shttps://github.com/<org>/<repo>/actions/runs/<run-id>/jobs/<number>%[1]s.
```

Reply:
That sounds like a sensible idea.